plugins/mcreator-localization/help/default/damagetype/normal_death_message.md

```diff
@@ -0,0 +1,7 @@
+This message is displayed in chat when a player dies from damage of this type, in most situations.
+
+You can use the following tokens in the message:
+- `<player>`: the name of the player who died
+- `<attacker>`: the name of the entity that dealt the damage
+
+NOTE: Use the second token only if the damage is always caused by a mob.
```

Comment:
> NOTE: Use the second token only if the damage is always caused by a mob.

Isn't the entity death message used in this case and thus attacker token is not used in this one?

Reply:
I see. I somehow thought "player death message" covered other entities, which is of course not correct conclusion from my side. All good then :)